test/test.js

```diff
@@ -180,19 +175,65 @@ describe('YeldaChat', () => {
     })
   })
 
+  describe('yeldaChat.voiceFirstUI', () => {
+    it('should contain voiceFirstUI class', () => {
+      // Reset the DOM
+      yeldaChat.unLoadChat()
+
+      // resets previous response
+      mock.reset()
+      // Mock JSON response
+      mock.get('https://app.yelda.ai//assistants/12345678/chatBubble/fr_FR', {
+        status: 201,
+        body: JSON.stringify({ data: {isVoiceFirstUI: true} })
+      })
+
+      const mockData = {
+        'assistantSlug': 'testClient',
+        'assistantId': '12345678',
+        'chatPath': 'chat'
+      }
+
+      yeldaChat.setupChat(mockData)
+      // setTimeout used to wait the for the mock response
+      setTimeout(() => {
+        expect(yeldaChat.iframeContainer).to.have.attribute('class', 'yelda_iframe_container voiceFirstUI y_active')
+      })
+    })
```

Comment:
Are you sure this test is working ?
I commented `        // this.iframeContainer.classList.add('voiceFirstUI')` and the test still pass whereas it should fail.
![image](https://user-images.githubusercontent.com/5340081/97546923-6ba68f80-19cd-11eb-84de-5adfb87ec687.png)

Reply:
Are you sure, you build the code before running the test command ?
npm run prepack will do both build and test

Reply:
haha indeed, i was sure we were testing the injector directly, not the build one.
OK for me, thanks